com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -506,6 +506,7 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
 #endif
             using (new ProfilingScope(null, Profiling.Pipeline.beginCameraRendering))
             {
+                UpdateCameraStereoMatrices(baseCameraData);
```

Comment:
I would suggest to call UpdateCameraStereoMatrices outside of the beginCameraRendering profiling scope.

Reply:
Good point, done!